lib/plugins/urls.js

```diff
@@ -5,10 +5,14 @@ var addImport = require('./../add-import');
 var stripQuotes = require('./../strip-quotes');
 var mark = require('./../mark');
 
-module.exports = function(ast, reworkInstance) {
+function transform(ast, reworkInstance) {
   ast.rules.forEach(function(rule) {
-    if (!rule.declarations)
+    if (!rule.declarations) {
+      if (rule.type === 'media') {
+        process(rule, reworkInstance);
```

Comment:
You are correct. That’s what I get for writing the code locally and then modifying in githubs editor.